nightly/jre-16/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM adoptopenjdk/openjdk16:debianslim-jre
```

Comment:
What was the motivation to switch from `openjdk` repository? lack of "jre-slim" version?

Reply:
I found following statement that suggested to look elsewhere:
> Starting with openjdk:12 the default image as well as the -oracle and -oraclelinux7 variants are based on the official Oracle Linux 7 image which is provided under the GPLv2 as per the Oracle Linux End User Agreement (EULA).
> 
> The OpenJDK binaries in the default image as well as the -oracle and -oraclelinux7 variants are built by Oracle and are sourced from the OpenJDK community. These binaries are licensed under the GPLv2 with the Classpath Exception.

see https://hub.docker.com/_/openjdk?tab=description&page=1&ordering=last_updated&name=jre

Reply:
Understood, makes sense.